### USER Input 
we use input function 

In [2]:
name=input("type you name")

In [3]:
print("Hello",name)

Hello zahoor


##### Note: 
            input function always take string  from user side

In [4]:
age=input("what is you age")

In [6]:
print("your age is "+ age)

your age is 24


In [9]:
# int function
number_one=input("enter first number ")
number_two=input("enter second number")
total=number_one + number_two
print(total)


3434


I have enter 34 first and second but i get 3434 because input take strint from user side

In [12]:
# how do we get int from user side
number_one=int(input("Enter first number")) 
number_two=int(input('Enter second  number')) 
total=number_one + number_two
print(total)
# str() 4 o/p "4", int 4 o/p 4 and  float 4 o/p 4.0


68


In [16]:
print("hello " + name + " your age is " ,int(age))

hello zahoor your age is  24


### Multiple values

In [19]:
# first method 
name=input('Enter your name: ')
age = input('Enter you  age: ')
print('Hello ' + name + " your age is " + age)

Hello Zahoor  your age is 24


In [25]:
# Second method 
name, age = input("Enter  your name  and age").split()
print('Hello ' + name + " your age is " + age)
# First enter your name and  space then enter age value
# You can also use what you want instead of space splite(','/'-'/'+'... so on)  

Hello Zahoor your age is 34
